plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/IDeployImagePageModel.java

```diff
@@ -94,7 +94,17 @@
 	 * @param imageName the name of the Docker Image to use
 	 */
 	void setImageName(String imageName);
-	
+
+	/**
+	 * Since method setImageName(String) ignores attempts to set an empty value, 
+	 * ui gets not synchronized with model as soon as image name is cleared.
```

Comment:
what I see is that the ObservableValue for the value is caching the old value, the value that existed before clearing it. The firing in the model (with oldValue=null) happens, but the observable for the widget wont change anything in the widget because the old value and the new one are equal. That's imho another issue in databinding.